README.md

```diff
@@ -1,38 +1,86 @@
-Jitsi Meet Electron
-====
-Electron application for [Jitsi Meet](https://github.com/jitsi/jitsi-meet).
+# Jitsi Meet Electron
 
-## Configuration
-You can change the Jitsi Meet deployment domain with the jitsiMeetDomain property from config.js
+Desktop application for [Jitsi Meet] built with [Electron].
+
+## Features
+
+- Works with any Jitsi Meet deployment
+- Local settings
+- Builtin auto-updates
+- Remote control
+- Always-On-Top window
+
+## Installation
+
+Grab the [latest release] and you're off to the races! The supported platforms
+are macOS, Windows (both 32 and 64bits) and GNU/Linux (64bits only).
+
+## Development
+
+If you want to hack on this project, here is how you do it.
+
+#### Building the sources
 
-## Building the sources
 ```bash
 npm install
 ```
 
-## Working with the [jitsi-meet-electron-utils](https://github.com/jitsi/jitsi-meet-electron-utils) sources
-By default the jitsi-meet-electron-utils is build from its git repository sources. The default dependency path in package.json is :
+#### Starting in development mode
+
+```bash
+npm start
+```
+
+#### Building the production distribution
+
+```bash
+npm run dist
+```
+
+#### Working with jitsi-meet-electron-utils
+
+[jitsi-meet-electron-utils] is a helper package which implements many features
+such as remote control and the always-on-top window. If new features are to be
+added / tested, running with a local version of these utils is very handy, here
+is how to do that.
+
+By default the jitsi-meet-electron-utils is build from its git repository
+sources. The default dependency path in package.json is:
+
 ```json
 "jitsi-meet-electron-utils": "jitsi/jitsi-meet-electron-utils"
 ```
 
 To work with local copy you must change the path to:
+
 ```json
 "jitsi-meet-electron-utils": "file:///Users/name/jitsi-meet-electron-utils-copy",
 ```
 
-To make the project you must force it to take the sources as `npm update` will not do it.
+To build the project you must force it to take the sources as `npm update` will
+not do it.
+
 ```bash
 npm install jitsi-meet-electron-utils --force
 node_modules/.bin/electron-rebuild
```

Comment:
`electron-rebuild` has been removed and `electron-builder install-app-deps` is automatically run `postinstall`.
This can be removed.

Reply:
Thanks for catching that!